CONTRIBUTING.adoc

```diff
@@ -85,6 +111,7 @@ proper code formatting. If you don't use a tool that supports editorconfig be
 sure to configure your editor to use the settings equivalent to our
 `.editorconfig` file.
 
+[#commit]
```

Comment:
I can't comment directly on the line in question, so getting it as close as I can.  We should probably remove the `--global` from the following:

```
$ git config --global user.name "J. Random User"
$ git config --global user.email "j.random.user@example.com"
```
It's possible that people may have more than one email account they use for contributions like a work email and a personal one both configured on the same computer.  It's safer to leave off the flag entirely which only sets the properties locally for the current repo.

Reply:
We probably also want to add a blurb like the following somewhere in this section to make the purpose of the individual contributor license more obvious:

`Signing off on a commit is agreeing to our {icla} and attesting that you either wrote the code or have the rights to contribute it.  The name and email address must be valid as we cannot accept anonymous contributions.`

Reply:
That's a useful callout.